src/database/index.tsx

```diff
@@ -252,72 +212,18 @@ export const saveScenario = async (
         }),
       );
 
-      console.log("create payload payload: " + JSON.stringify(payload));
+      const scenarioDocRef = await db
+        .collection(scenariosCollectionId)
+        .add(payload);
 
-      scenarioDocRef = await db.collection(scenariosCollectionId).add(payload);
+      scenarioId = scenarioDocRef.id;
     }
 
-    return scenarioDocRef;
+    return await getScenario(scenarioId);
   } catch (error) {
     console.error("Encountered error while attempting to save a scenario:");
     console.error(error);
-  }
-};
-
-export const getBaselineScenario = async (): Promise<Scenario | null> => {
-  const baselineScenarioRef = await getBaselineScenarioRef();
-
-  if (!baselineScenarioRef) return null;
-
-  const result = await baselineScenarioRef.get();
-  let scenario: Scenario = result.data() as Scenario;
-  scenario.id = baselineScenarioRef.id;
-
-  // NOTE: We should be able to remove this check and save once this issue
-  // is resolved: https://github.com/Recidiviz/covid19-dashboard/issues/186
```

Comment:
I removed this code as part of this commit because I manually migrated the existing scenarios to ensure that they all have the `promoStatuses` map.  I also removed the legacy `showPromo` flag for all Scenarios in Firestore

Reply:
Thanks for taking care of this, @sprad! I and Daniela thank you.